scripts/validateDeploymentAndGenesis.js

```diff
@@ -137,7 +137,7 @@ module.exports = async () => {
     genesisConfig.teamGrants.forEach(async grant => {
         const vestingHolderAddr = await genesisManager.vestingHolders.call(grant.receiver)
         const vestingHolder = await TokenVesting.at(vestingHolderAddr)
-        assert.equal(await vestingHolder.beneficiary.call(), grant.receiver, "should be correct vesting grant receiver")
+        assert.equal((await vestingHolder.beneficiary.call()).toLowerCase(), grant.receiver.toLowerCase(), "should be correct vesting grant receiver")
```

Comment:
This assertion could fail before if the addresses entered in `genesis.config.js` were checksummed addresses. Now the assertion compares the lower cased representation of the input addresses